docs/release-guide/README.md

```diff
@@ -3,13 +3,14 @@
 This is the process we follow when releasing a new version of Cesium for Omniverse on GitHub.
 
 1. [Release a new version of Cesium for Omniverse Samples](#releasing-a-new-version-of-cesium-for-omniverse-samples).
+2. Update [`extension.py`](../../apps/exts/cesium.performance.app/cesium/performance/app/extension.py) in the Performance App with the latest public `ION_ACCESS_TOKEN`.
```

Comment:
This is similar to step 10. Can you update the wording in step 10 and remove this step?